src/commands/LettuceMeetVisualizer.ts

```diff
@@ -0,0 +1,274 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, EmbedFieldData, MessageEmbed } from 'discord.js';
+import Command from '../Command';
+import { BotClient } from '../types';
+import axios from 'axios';
+import Logger from '../utils/Logger';
+import { DateTime } from 'luxon';
+
+// URL for Lettuce Meet GraphQL API, and the query to get the event data.
+const URL = 'https://api.lettucemeet.com/graphql';
+const query = `query EventQuery($id: ID!) {
+  event(id: $id) {
+    ...Event_event
+  }
+}
+fragment Event_event on Event {
+  title
+  pollResponses {
+    user {
+      ...on AnonymousUser {
+          name email
+      }
+      ...on User {
+          name email
+      }
+    }
+    availabilities {
+      start end
+    }
+  }
+}`;
+const LETTUCE_MEET_INTERVAL = 30;
+
+/**
+ * Data returned by the Lettuce Meet GraphQL API.
+ */
+interface BoardData {
+  data: {
+    event: {
+      title: string;
+      pollResponses: {
+        user: {
+          name: string;
+          email: string;
+        };
+        availabilities: UserAvailability[];
+      }[];
+    };
+  };
+}
+
+/**
+ * Availability of a user.
+ */
+interface UserAvailability {
+  start: string;
+  end: string;
+}
+
+/**
+ * Availability data for a time block
+ */
+interface AvailabilityData {
+  numAvailable: number;
+  peopleAvailable: string[];
+}
+
+
+/**
+ * Visualizes the Lettuce Meet board.
+ */
+export default class LettuceMeetVisualizer extends Command {
+  constructor(client: BotClient) {
+    const definition = new SlashCommandBuilder()
+      .setName('lettucemeet')
+      .addStringOption((option) =>
+        option.setName('code').setDescription('Lettuce Meet board code (code at the end of url)')
+          .setRequired(true))
+      .addStringOption((option) =>
+        option.setName('users').setDescription('Emails or names of the users to filter by').setRequired(false))
```

Comment:
Maybe consider making this `Emails or names of the users to include in the results`? to filter by is kinda unclear on what it's doing